src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  text-transform: uppercase;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin: 10px 50px;
+}
+
+.nav {
+  margin-right: 50px;
```

Comment:
That's better to use paddings for the header instead of margins for the navigation bar